pkg/utils/logger/logger.go

```diff
@@ -68,3 +69,12 @@ func getLogLevel() string {
 
 	return seelogLevel.String()
 }
+
+func getLogOutput(logFilePath string) string {
+	switch strings.ToLower(os.Getenv(envLogOutput)) {
```

Comment:
What about just reusing `AWS_VPC_K8S_CNI_LOG_FILE` and adding the environment variable to the documentation? This would just become:
```
    switch logFilePath {
	case "stdout":
		return `<console />`
	default:
return fmt.Sprintf(`<rollingfile filename="%s" type="date" datepattern="2006-01-02-15" archivetype="none" maxrolls="24" />`, logFilePath)
	}
```

And something like

`AWS_VPC_K8S_CNI_LOG_FILE`
Type: String  
Default: Unset
Valid Values: `stdout` or a file path
Specifies where to write the logging output. Either to `stdout` or to override the default file.

in the README.md?

Reply:
I completely agree with you. Updated!